### PR TITLE
Updates

### LINKED ISSUE #10: The latest version is not compatible with Nautobot 1.3.2
### Environment
* Python version:  3.7
* Nautobot version:  1.3.2
* nautobot-ssot-servicenow version:  0.9.1

### Expected Behavior
Compatibility between the plugin and the latest Nautobot version

### Observed Behavior
The plugin depends on the following PyYAML library version: `nautobot-ssot-servicenow (0.9.1) depends on PyYAML (>=5.4.1,<6.0.0)`
Nautobot requires the PyYAML version (>=6.0,<6.1).

### Steps to Reproduce
1. Install Nautobot 1.3.2 and this plugin of version 0.9.1

### PR DESCRIPTION
**Pull request by ubajze**
Fixes #10

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -19,12 +19,19 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
-nautobot = "^1.2.0"
+python = "^3.7"
+nautobot = "1.3.4"
```
[the remark this answers is not in the document]
Comment (ubajze) [reply pyproject.toml]: @glennmatthews Can you please check the updated file? Is it ok like that?

File: pyproject.toml [hunk 2]
```diff
@@ -19,12 +19,19 @@ packages = [
...
+Jinja2 = "*"
+#pysnow = "^0.7.17"
+# We are installing pysnow from our own fork.
+# https://github.com/networktocode-llc/pysnow
+# After this PR is merged, we can go back to the
+# official release:
+# https://github.com/rbw/pysnow/pull/186
+# (dependency issue with pytz)
+pysnow = {file = "packages/pysnow-2022.5.25-py2.py3-none-any.whl" }
+PyYAML = "*"
```
[the remark this answers is not in the document]
Comment (ubajze) [reply pyproject.toml]: @glennmatthews Can you please check the updated file? Is it ok like that?

File: pyproject.toml [hunk 3]
```diff
@@ -19,12 +19,19 @@ packages = [
...
-PyYAML = "^5.4.1"
+Jinja2 = "*"
+#pysnow = "^0.7.17"
+# We are installing pysnow from our own fork.
+# https://github.com/networktocode-llc/pysnow
+# After this PR is merged, we can go back to the
+# official release:
+# https://github.com/rbw/pysnow/pull/186
+# (dependency issue with pytz)
+pysnow = {file = "packages/pysnow-2022.5.25-py2.py3-none-any.whl" }
```
[the remark this answers is not in the document]
Comment (ubajze) [reply pyproject.toml]: @glennmatthews Can you please check the updated file? Is it ok like that?

[the remark this answers is not in the document]
Comment (glennmatthews) [reply pyproject.toml]: No, doesn't work for the wheel:

```
% python3 -m venv venv
% source venv/bin/activate
(venv) % pip3 install dist/nautobot_ssot_servicenow-0.9.1-py3-none-any.whl 
Processing ./dist/nautobot_ssot_servicenow-0.9.1-py3-none-any.whl
ERROR: Exception:
...
pip._vendor.pkg_resources.RequirementParseError: Invalid URL: nautobot_ssot_servicenow/third-party/pysnow-2022.5.25-py2.py3-none-any.whl
```

For this to be distributable, we'll need the actual pysnow source under third-party, not just a wheel file. I'll see if I can make the appropriate update.

File: pyproject.toml (hunk 1 above)
[the remark this answers is not in the document]
Comment (glennmatthews) [reply pyproject.toml]: `^1.2.0` would be correct still in that case.

**Comment by glennmatthews** [comment]
Superseded by #15.

**Pull request closed without merging**